app/Http/Resources/StatusResource.php

```diff
@@ -48,6 +48,10 @@ public function toArray($request): array {
                 'manualArrival'       => $this->trainCheckin->manual_arrival?->toIso8601String(),
                 'origin'              => new StopoverResource($this->trainCheckin->origin_stopover),
                 'destination'         => new StopoverResource($this->trainCheckin->destination_stopover),
+                'operator'            => [
```

Comment:
Please convert this to a nullable OperatorResource.

Reply:
How do I do that? 🙃 (I have no idea using PHP)

Reply:
Ok, I tried and tested but maybe there is a better way

Reply:
You can create resources in Laravel with `php artisan make:resource OperatorResource`.

Then just have a look at the implementation of StopoverResource. :)

Reply:
Ok I guess then it's the better way than using `is_null(..) ? null : [...]` :D

Reply:
Exactly. :D